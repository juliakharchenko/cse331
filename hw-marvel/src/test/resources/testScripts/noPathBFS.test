# Tests finding a path between two nodes that don't have a path.

LoadGraph nba nbaPlayers.csv

FindPath nba MICHAEL-JORDAN REGGIE-MILLER