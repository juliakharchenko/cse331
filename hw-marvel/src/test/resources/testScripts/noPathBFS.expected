# Tests finding a path between two nodes that don't have a path.

loaded graph nba

path from MICHAEL-JORDAN to REGGIE-MILLER:
no path found